pkg/cmdimpl/add_test.go

```diff
@@ -171,17 +171,75 @@ func handleGitLsRemote(arglist ...interface{}) ([]byte, []byte, error) {
 	return nil, nil, fmt.Errorf("NO!")
 }
 
-var fakeGitClient = gitfakes.FakeGit{}
+var failGitClient = gitfakes.FakeGit{
+	CloneStub: func(ctx context.Context, a, b, c string) (bool, error) {
```

Comment:
a failure would probably be returning an error instead of exiting the process.

Reply:
This is so that any use of the code anywhere downstream will fail the test w/o trying to propagate the failure to ensure dry run is working. Returning an error wouldn't work.

Reply:
the use of invalid/non-existent repo addresses would probably accomplish the same, without requiring you to implement those, but I'm fine with it. It's up to you.

Reply:
That makes sense; I was just trying to be consistent across all the overrides and that was the simplest way. Not wedded to it :-)